src/main/java/com/google/sps/servlets/NewRequestServlet.java

```diff
@@ -85,36 +84,37 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
     
     // initialize firebase app 
-    FirebaseApp defaultApp = initializeFirebaseApp();
+    FirebaseApp firebaseApp = FirebaseSingletonApp.getInstance();
```

Comment:
Please write an unit test with a fake firestore to verify that your code is working after (and before) your changes.

https://firebase.google.com/docs/rules/unit-tests